nemo/collections/asr/models/asr_model.py

```diff
@@ -47,3 +51,61 @@ def multi_test_epoch_end(self, outputs, dataloader_idx: int = 0):
         wer_denom = torch.stack([x['test_wer_denom'] for x in outputs]).sum()
         tensorboard_logs = {'test_loss': val_loss_mean, 'test_wer': wer_num / wer_denom}
         return {'test_loss': val_loss_mean, 'log': tensorboard_logs}
+
+    def export(
+        self,
+        output: str,
+        input_example=None,
+        output_example=None,
+        verbose=False,
+        export_params=True,
+        do_constant_folding=True,
+        keep_initializers_as_inputs=False,
+        onnx_opset_version: int = 12,
+        try_script: bool = False,
+        set_eval: bool = True,
+        check_trace: bool = True,
+        use_dynamic_axes: bool = True,
+    ):
+        qual_name = self.__module__ + '.' + self.__class__.__qualname__
+        output1 = os.path.join(os.path.dirname(output), 'encoder_' + os.path.basename(output))
+        output1_descr = qual_name + ' Encoder exported to ONNX'
+        if input_example is None:
+            input_example = self.encoder.input_example()
+
+        encoder_onnx = self.encoder.export(
+            output1,
+            input_example=input_example,
+            output_example=None,
+            verbose=verbose,
+            export_params=export_params,
+            do_constant_folding=do_constant_folding,
+            keep_initializers_as_inputs=keep_initializers_as_inputs,
+            onnx_opset_version=onnx_opset_version,
+            try_script=try_script,
+            set_eval=set_eval,
+            check_trace=check_trace,
+            use_dynamic_axes=use_dynamic_axes,
+        )
+
+        output2 = os.path.join(os.path.dirname(output), 'decoder_' + os.path.basename(output))
+        output2_descr = qual_name + ' Decoder exported to ONNX'
+        decoder_onnx = self.decoder.export(
+            output2,
+            input_example=None,
+            output_example=output_example,
+            verbose=verbose,
+            export_params=export_params,
+            do_constant_folding=do_constant_folding,
+            keep_initializers_as_inputs=keep_initializers_as_inputs,
+            onnx_opset_version=onnx_opset_version,
+            try_script=try_script,
+            set_eval=set_eval,
+            check_trace=check_trace,
+            use_dynamic_axes=use_dynamic_axes,
+        )
+
+        output_model = attach_onnx_to_onnx(encoder_onnx, decoder_onnx, "DC")
```

Comment:
please use something unique here which is similar to class name like "ASRM" 
DC most probably taken and if someone would merge this onnx it's theoretically possible to get names clashed.

Reply:
Well, we do not do multiple-level models yet, and actually, the collision would be guaranteed if we try to concat any 2 of the same models. A real fix, I think,  should be using "a." prefix on first and "b." prefix on second argument inside attach_onnx_to_onnx - no prefix arg needed.